subprojects/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/extensions/types/InstrumentedTypesResourceGenerator.java

```diff
@@ -44,12 +44,12 @@ public GenerationResult generateResourceForRequests(Collection<CallInterceptionR
         return new GenerationResult.CanGenerateResource() {
             @Override
             public String getPackageName() {
-                return "org.gradle.internal.instrumentation";
+                return "";
```

Comment:
❓ Why change this? It seems prudent to use a package name for our generated classes. Or is this because we put the file in `META-INF` instead of under a package?

Reply:
Yes, this is because we put it in to the META-INF, before this resource was next to classes in `org.gradle.internal.instrumentation` package. I think it makes more sense to have it in META-INF, or you feel other way?

Reply:
I don't feel strongly about where we put it. `META-INF` seems fine, too.